.github/workflows/build-push-collections.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Collections
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+env:
+  PUSH_BRANCH: "main"
+
+jobs:
+  build-push-collection:
+    permissions:
+      contents: write
+      packages: write
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setting Workflow Variables
+        id: set-workflow-variables
+        run: |
+          PUSH_EVENT='false'
+
+          if [[ "${{ github.ref }}" =~ ^refs/tags/ ]]; then
+            PUSH_EVENT='true'
+          fi
+
+          BRANCH_NAME="$(echo ${GITHUB_REF#refs/heads/})"
+          if [ "$BRANCH_NAME" == "${{ env.PUSH_BRANCH }}" ]; then
+            PUSH_EVENT='true'
+          fi
+
+          echo "::set-output name=push_event::$PUSH_EVENT"
+
+        # Manually download client. To be replaced with dynamic downloader next client release
+      - name: Download Client
+        run: |
+          curl -Ls https://github.com/uor-framework/uor-client-go/releases/download/v0.3.0-alpha.1/uor-client-go-linux-amd64.tar.gz | tar xvz -f - -C /usr/local/bin
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
```

Comment:
Are we using qemu / buildx in later steps of this action?

Reply:
supports registry login

Reply:
is the qemu step required for login?

Reply:
Removed!